src/Facets/GnosisBridgeFacet.sol

```diff
@@ -46,16 +51,24 @@ contract GnosisBridgeFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
         doesNotContainDestinationCalls(_bridgeData)
         validateBridgeData(_bridgeData)
```

Comment:
I think there are some modifiers missing (nonReentrant, doesNotHaveSourceSwaps, etc.)
Might be because this facet is so old.